music_vae/src/model.ts

```diff
@@ -724,7 +725,7 @@ class MusicVAE<A extends magenta.controls.ControlSignalUserArgs> {
       let inputTensors = tf.stack(inputSequences.map(
           t => this.dataConverter.toTensor(t) as tf.Tensor2D)) as tf.Tensor3D;
 
-      if (this.controlSignal) {
+      if (!isNullOrUndefined(this.controlSignal)) {
```

Comment:
What's different about this check?

Reply:
Nothing in this case. I tried this when I wasn't sure of the original problem. Reverted.